source/env_tutorial/pettingzoo_zh.rst

```diff
@@ -0,0 +1,316 @@
+Petting

```

Comment:
add this file into `source/env_tutorial/index_zh.rst` to ensure correct strcuture